packages/utils/README.md

```diff
@@ -0,0 +1,15 @@
+# @noaignite/utils
+
+Utils - Utility functions.
+
+## Installation
+
+Utils is available as an [npm package](https://www.npmjs.com/package/@noaignite/utils).
+
+```sh
+// with npm
+npm install @noaignite/utils
+
+// with yarn
+yarn add @noaignite/utils
+```
```

Comment:
Should we omit mentions of `npm` in favor of `pnpm` or do we just add `pnpm` here.
@maeertin @adamsoderstrom